Mathlib/Geometry/Manifold/AnalyticManifold.lean

```diff
@@ -166,27 +111,78 @@ theorem symm_trans_mem_analyticGroupoid (e : PartialHomeomorph M H) :
 instance : ClosedUnderRestriction (analyticGroupoid I) :=
   (closedUnderRestriction_iff_id_le _).mpr
     (by
-      rw [StructureGroupoid.le_iff]
+      apply StructureGroupoid.le_iff.mpr
       rintro e ⟨s, hs, hes⟩
       apply (analyticGroupoid I).mem_of_eqOnSource' _ _ _ hes
       exact ofSet_mem_analyticGroupoid I hs)
 
+/-- `f ∈ analyticGroupoid` iff it and its inverse are analytic within `range I`. -/
+lemma mem_analyticGroupoid {I : ModelWithCorners 𝕜 E H} {f : PartialHomeomorph H H} :
```

Comment:
Should this be `mem_analyticGroupoid_iff`? I'm not sure; a final maintainer can weigh in.

Reply:
Also: should this marked simp? Perhaps not, but I'd like to hear another opinion on this.

Reply:
A bunch of lemmas were recently changed to remove the `iff`, so I'll leave that for someone else. I don't think it should be simp: the expansion is too big.